packages/zundo/src/index.ts

```diff
@@ -68,3 +68,4 @@ const zundoImpl: ZundoImpl = (config, baseOptions) => (set, get, _store) => {
 };
 
 export const temporal = zundoImpl as unknown as Zundo;
+export { TemporalStateWithInternals };
```

Comment:
Is this export needed?

Reply:
ideally we should not be exposing that, its currently sitting here for testing purposes